### PR TITLE
feat(sqlite): add `fileMustExist` option.

### PR DESCRIPTION
**Pull request by r-rajaneesh**
➕ Added a `fileMustExist` in the database options

### What does this PR do?

This PR adds a check from `better-sqlite3` `fileMustExist` in the options of the constructor.
If `fileMustExist` is set to `true` then it'll not create a file on default else the file is created

- [ ] Documentation or TypeScript types (it's okay to leave the rest blank in this case)
- [ ] Code changes

### How did you verify your code works?
Needed review of this before doing tests.

### CODE REVIEW

**Comment by gvilums** [comment]
Thank you for the PR. Do I understand correctly that this new option is the inverse of the `create` option? In that case I don't think we should implement it, as, in my opinion, the convenience of these redundant options is negated by the confusion they cause.

**Comment by r-rajaneesh** [comment]
> Thank you for the PR. Do I understand correctly that this new option is the inverse of the `create` option? In that case I don't think we should implement it, as, in my opinion, the convenience of these redundant options is negated by the confusion they cause.

@gvilums, Jarred and Dave told me to create an issue for this, since `better-sqlite3` has this options and **Bun** does not, even tho it says it implements all the feature in `better-sqlite3`. But I created a PR instead, and `create` was never an option in `better-sqlite3` in the first place.

File: src/js/bun/sqlite.ts [hunk 1]
```diff
@@ -187,6 +187,12 @@ class Database {
     if (typeof options === "object" && options) {
       flags = 0;
 
+      if(!options.fileMustExist) {
+        options.create = true;
+      } else {
+        options.create = false;
+      }
```
Comment (Electroid) [inline src/js/bun/sqlite.ts]:
```suggestion
      if(options.fileMustExist) {
        options.create = false;
      }
```

**Review by Electroid — CHANGES_REQUESTED** [review]

File: src/js/bun/sqlite.ts (hunk 1 above)
Comment (Jarred-Sumner) [reply src/js/bun/sqlite.ts]:
```suggestion
      if(typeof options.fileMustExist === "boolean") {
          options.create = !options.fileMustExist;
      }
```

Comment (paperclover) [reply src/js/bun/sqlite.ts]: this likely fixes the test failure

**Comment by Jarred-Sumner** [comment]
Closing due to lack of tests

**Pull request closed without merging**